library/alpine/3.21/Makefile

```diff
@@ -0,0 +1,31 @@
+# This file is generated by the template.
+
+REGISTRY?=lcr.loongnix.cn
+ORGANIZATION?=library
+REPOSITORY?=alpine
+TAG?=3.21
+REL=3
+
+IMAGE=$(REGISTRY)/$(ORGANIZATION)/$(REPOSITORY):$(TAG)
+
+ROOTFS=rootfs.tar.gz
+
+ALIAS=lcr.loongnix.cn/library/alpine:${TAG}-temporary
+
+default: image
+
+image: $(ROOTFS)
+	docker build \
+		--build-arg ROOTFS=$(ROOTFS) \
+		-t $(IMAGE) \
+		.
+
+push: alias
+	docker push $(IMAGE)
+	docker push $(ALIAS)
+
+$(ROOTFS):
+	wget -O $(ROOTFS) https://dl-cdn.alpinelinux.org/alpine/v${TAG}/releases/loongarch64/alpine-minirootfs-${TAG}.${REL}-loongarch64.tar.gz
```

Comment:
3.21 is the first release of Alpine with official support, so I've used the release URL (`/v${TAG}/` instead of `/edge/`).